src/debug/ee/controller.cpp

```diff
@@ -69,7 +69,7 @@ bool DebuggerControllerPatch::IsSafeForStackTrace()
 
 }
 
-#ifndef _TARGET_ARM_
+#if !(defined(_TARGET_ARM_) || (defined(_TARGET_ARM64_) && defined(PLATFORM_UNIX)))
```

Comment:
This condition is repeated many times. Would it make sense to create a new defined for this?

Reply:
I'll add `FEATURE_EMULATE_SINGLESTEP`

Reply:
Isn't that counter intuitive? This is is the define that asks if single step is hardware supported.

Reply:
Counter intuitive? We are adding support for `arm64 single step emulation` why would `FEATURE_EMULATE_SINGLESTEP` be counter-intuitive as a feature name.

It obviously has to be used correctly as needed
```C++
#ifdef FEATURE_EMULATE_SINGLESTEP
```
```C++
#ifndef FEATURE_EMULATE_SINGLESTEP
```

Reply:
Done

Reply:
Oh ok, so the positive of the condition. Thanks steve :)